### PR TITLE
:memo: Rewrite the Docker licencing instructions

### PR DESCRIPTION
**Pull request by jasonBirchall**
## 👀 Purpose

- This pull request aims to revise the internal documentation regarding DockerHub and Docker Desktop usage at the Ministry of Justice. The goal is to make the document more structured, clear, and informative so that all developers know the processes and requirements for using Docker in their work.

## ♻️ What's changed

- Improved the overall structure and wording for better comprehension and flow.

## 📝 Notes

-

### CODE REVIEW

File: source/documentation/services/dockerhub.html.md.erb [hunk 1]
```diff
@@ -1,40 +1,45 @@
...
-## Support
+## Logging In
 
-If you are having trouble completing sign-in with SSO in Docker Desktop, note that it requires version 4.4.2 at minimum for the process to work.
+To log in, access DockerHub via [this link](https://hub.docker.com/) or through the Docker Desktop app. Choose 'GitHub' for SSO authentication.
 
-If you have any issues please contact #ask-operations-engineering slack channel.
+## Support and Troubleshooting
+
+- Docker Desktop requires version 4.4.2 or later for SSO compatibility.
```
Comment (tamsinforbes) [inline source/documentation/services/dockerhub.html.md.erb:44]: version 4.4.2 of what? Docker Desktop? Docker? DockerHub? SSO?

**Review by tamsinforbes — CHANGES_REQUESTED** [review]
Small clarification on what version 4.4.2 refers to please? Unless I've misread?

**Review by AntonyBishop — DISMISSED** [review]

File: source/documentation/services/dockerhub.html.md.erb (hunk 1 above)
Comment (jasonBirchall) [reply source/documentation/services/dockerhub.html.md.erb:44]: Thanks for the review, @tamsinforbes. Do you have an example of the suggested change? To me, "Docker Desktop requires version 4.4.2..." is clear.

Comment (jasonBirchall) [reply source/documentation/services/dockerhub.html.md.erb]: Cool, done.

**Review by tamsinforbes — APPROVED** [review]
🌝🌖🚀LGTM🚀🌘🌚

**Pull request merged**